src/editor/editor-mathfield.js

```diff
@@ -3264,132 +3264,126 @@ MathField.prototype.$setConfig = function(conf) {
     }
 }
 
-
-
-
-MathField.prototype._speakWithSynchronizedHighlighting = function(text) {
-    if (!this.config.handleReadAloud) return;
-    this.config.handleReadAloud(this.field, text, this.config);
-}
-
-
-MathField.prototype._speak = function(text) {
-    if (!this.config.handleSpeak) return;
-    this.config.handleSpeak(text, this.config);
-}
-
 /**
- * @method MathField#speakSelection_
+ * 
+ * Speak some part of the expression, either with or without synchronized highlighting.
+ * 
+ * @param {string} amount (all, selection, left, right, group, parent)
+ * @param {boolean} withHighlighting - If true, synchronized highlighting of speech will happen (if possible)
```

Comment:
Use a named param rather than a boolean, so that the call can be `speak_("all", {withHighlighting: true})` rather than `speak("all", true)` and provide a default value for `withHighlighting`.